README.md

```diff
@@ -1 +1,51 @@
 # Modbus Processing Unit (MPU)
+
+Provides a common modbus communication library, which can be left running in
+FPGA and provide telemetry. The major idea is to create in FPGA a simple CPU (a
+Turing Machine if you want), narrowly designed into reading out data from
+Modbus devices. The design is:
```

Comment:
Please consider to do something like this to improve the maintainability (the similar thing for other places):

```
Provides a common modbus communication library, which can be left running in
FPGA and provide telemetry.
The major idea is to create in FPGA a simple CPU (a
Turing Machine if you want), narrowly designed into reading out data from
Modbus devices.
The design is:
```

Reply:
I don't believe that will significantly improve maintainability. I re-format my free text in Vim after editing, this will make the whole process just messier.